### ISSUE TITLE
mapping <C-d> as delete

### ISSUE THREAD

**Issue by hnsol** [esm7/obsidian-vimrc-support#8]
Thank you great Plug-in, I really love to use it ! ... but I'm stuck with one problem.

Is it possible to map `<C-d>` as delete in insert mode ? 

`imap <C-d> <Del>` ... not works

`imap <C-d> <Esc>lxi`  ... almost OK, however not good at Begging / End of line.

Is any appropriate setting or workaround?

**Comment by esm7** [comment]
Seems like `imap` with the Ctrl key doesn't work at all.
Also seems like this is a CodeMirror bug, since it also happens in their online Vim demo: https://codemirror.net/demo/vim.html
Wish I could help further, but I think that the only way to go is to open a bug [in their repo](https://github.com/codemirror/CodeMirror/issues).

**Comment by hnsol** [comment]
thank you @esm7 . I'll try.

**Comment by hnsol** [comment]
I found a workaround.

Add a line at 
.obsidian/plugins/obsidian-vimrc-support/main.js

like below:

```
                // Make sure that we load it just once per CodeMirror instance.
                // This is supposed to work because the Vim state is kept at the keymap level, hopefully
                // there will not be bugs caused by operations that are kept at the object level instead
                CodeMirror.Vim.unmap('<C-d>', 'insert');
                //
                CodeMirror.Vim.loadedVimrc = true;
            }
```

Not beautiful way ... but it works.
Thank you for your kindly support!

**Comment by esm7** [comment]
Does this have a different effect than executing `unmap <C-d>` (interactively or via your vimrc)?

**Comment by hnsol** [comment]
Actually, `CodeMirror.Vim.unmap()` acts differently. When executing `unmap <C-d>` (both interactively **and** via `.obsidian.vimrc`):

- normal mode: unmapped (never 'moveByScroll')
- insert mode: not unmapped (still 'indentRight')

Also, when I tested at CodeMirror's online Vim demo (https://codemirror.net/demo/vim.html , of course interactively), I got same results (normal mode: unmapped, insert mode: not unmapped).

So it must be a CodeMirror bug. Maybe due to https://codemirror.net/keymap/vim.js, however, I cannot follow this well.
